spec/support/step_nav_steps.rb

```diff
@@ -187,6 +187,14 @@ def then_I_see_the_new_step_by_step_page
     expect(page).to have_content("How to bake a cake")
   end
 
+  def then_I_can_submit_the_step_by_step_for_2i_review
+    within(".app-side__actions") do
+      expect(page).to have_link("Submit for 2i review")
+    end
+  end
+
+  alias_method :and_I_can_submit_the_step_by_step_for_2i_review, :then_I_can_submit_the_step_by_step_for_2i_review
```

Comment:
This alias method isn't needed at this point in the git history - can just go straight to `def and_I_can_submit_the_step_by_step_for_2i_review`.

Reply:
Removed this.